tools/pepsi/src/upload.rs

```diff
@@ -127,3 +170,19 @@ pub async fn upload(arguments: Arguments, repository: &Repository) -> anyhow::Re
 
     Ok(())
 }
+
+#[cfg(test)]
+mod test {
+
+    fn test() {
```

Comment:
- You need a `#[test]` in the line before to mark this function as a test.
- Please name test functions after the fact you are testing. Also apply DRY and don't write the word `test` as this is a test, we can see that. Your fact is probably something like that the extracted version matches the version of the os-release file.

Reply:
Ok

Reply:
Please be more on-point with the test case name. I mentioned earlier that you do not need to write comments everywhere because our goal is that the code documents itself. This requires names to be good. "Extracts version num" is not the fact you are testing. Test cases check one fact of e.g. a unit. One fact may need to be checked with one or multiple assertions. In your case, you have a single assertion which also gives you the fact and this should be expressed in the test case name. Please improve the name.

Reply:
I renamed the test to 'matches_stable_os_version'